rados/rados_test.go

```diff
@@ -296,6 +343,54 @@ func (suite *RadosTestSuite) TestMakeDeletePool() {
 	if found {
 		suite.T().Error("Deleted pool still exists")
 	}
+
+	pool, _ = suite.conn.GetPoolByName(new_name)
```

Comment:
assert for  error

Reply:
An error is expected, the pool should not exist anymore.